articles/tutorials/data-scenarios/index.md

```diff
@@ -0,0 +1,257 @@
+---
+title: User Data Storage Scenario
+description: Demonstrating the best practices in using Auth0 storage mechanisms through the scenario of a native Swift app with a Node API backend.
+url: /tutorials/user-data-storage-scenario
+---
+
+# Auth0 User Data Storage Scenario
+
+Auth0 provides multiple locations for storing different types of data associated with authenticating an app’s users. The purpose of this document is to demonstrate the best practices in using these storage mechanisms efficiently and securely. It also gives you a look at an example of the end-to-end experience of an application using auth0 and an external database. We created an application to illustrate the important distinctions a developer must make when storing their user data with Auth0. For this example case, we started with a basic mobile app for iOS (coded in Swift) from the Auth0 seed project for an [iOS mobile app](/quickstart/native/ios-swift). As a backend for the app, we used the Auth0 seed project for a simple [Node.js API](/quickstart/backend/nodejs). As we discuss the different types of data and the best places to store them, we will continue to use this application as the example case. See the [Mobile + API architecture scenario](/architecture-scenarios/application/mobile-api) from our documentation to give you a visual of how the application is structured. 
+
+## Table of Contents
+
+- [Where do I put my authentication data?](/tutorials/user-data-storage-scenario#where-do-i-put-my-authentication-data-)
+- [Why not put all the app’s data in the Auth0 data store?](/tutorials/user-data-storage-scenario#why-not-put-all-the-app-s-data-in-the-auth0-data-store-)
+- [When should I use the Auth0 data store?](/tutorials/user-data-storage-scenario#when-should-i-use-the-auth0-data-store-)
+- [Review](/tutorials/user-data-storage-scenario#review)
+
+
+## Where do I put my authentication data?
+
+Auth0 has a data store which serves as a way for developers to store data associated with their users’ profiles, beyond the basic information Auth0 uses for authentication, i.e name, email, username, password, and so on. You can also [Authenticate Users with Username and Password using a Custom Database](/connections/database/mysql) to store such information if your needs are different from what the Auth0 data store offers. However, we recommend storing authentication-related data in Auth0’s data store in order to make things less complicated by allowing you to easily manage user data through Auth0’s [dashboard](${uiURL}).
```

Comment:
@eharkins I'm not sure that it's clear what these *authentication data* are. Are we talking `user_metadata` and `app_metadata` here? Maybe add an example or even better a note with a link to the user profile structure doc?